cwe_checker_rs/src/analysis/backward_interprocedural_fixpoint/mod.rs

```diff
@@ -54,6 +52,8 @@ pub trait Context<'a> {
     ) -> Option<Self::Value>;
 
     /// Transition function for in-program calls.
+    /// The target value is comming in via the call edge from the called subroutine and
+    /// the return_value is comming in via the call stub edge from the return node of the subroutine
```

Comment:
it is coming from the returned-to site of the caller, right? Not from the returned-from site of the callee like in the forward analysis. This is why I am being so insistent on being precise here...